stdlib/public/core/UnboundCapacity.swift

```diff
@@ -82,8 +82,19 @@ extension UnboundCapacity {
 
   /// Converts our contents into an array with at least the given
   /// capacity, and presents it to `body` for mutation.
-  internal mutating func withMutableArray<R>(
-    minCapacity: Int = 0,
+  //
+  // Michael NOTE: temporarily public, should probably make private
+  //
+  // Michael NOTE: forwarding version added due to odd linker behavior. Linker
+  // couldn't find the version with a default argument.
+  //
+  public mutating func withMutableArray<R>(
+    body: (inout [Base.Iterator.Element]) throws->R
+  ) rethrows -> R {
+    return try withMutableArray(minCapacity: 0, body: body)
+  }
```

Comment:
My use of withMutableArray had sad FIXMEs around it. I would love to avoid these and use `withUnsafeMutableBufferPointer`! It doesn't currently exist, but I'll try adding it soon.

I do need to potentially resize the array if FCC triggers decompositions that cause us to overflow, but I can probably do that dance outside of a `withUnsafeMutableBufferPointer` call.